ani-cli

```diff
@@ -150,54 +199,69 @@ get_video_quality() {
 	printf '%s' "$video_link"
 }
 
-dep_ch () {
-	for dep; do
-		if ! command -v "$dep" >/dev/null ; then
-			die "Program \"$dep\" not found. Please install it."
-		fi
-	done
+
+###############
+# Text output #
+###############
+
+err () {
+	# display an error message to stderr (in red)
+	printf "$c_red%s$c_reset\n" "$*" >&2
 }
 
-# get query
-get_search_query () {
-	if [ -z "$*" ]; then
-		printf "$c_cyan%s" "Search Anime: "
-		read -r query
-	else
-		query=$*
-	fi
+inf () {
+	# display an informational message (first argument in green, second in magenta)
+	printf "$c_green%s $c_magenta%s$c_reset\n" "$1" "$2"
 }
 
-# create history file
-[ -f "$logfile" ] || : > "$logfile"
+prompt () {
+        # prompts the user with message in $1 and saves the input to the variables in $2 and $3 
+        printf "$c_blue%s: $c_reset" "$1"
+        read -r input1 input2
+        # indirectly assigning the input to the variable in $2 and in $3 if present
+        if [ -z "$3" ]; then
+                eval "$2=\$input1"
+        else
+                eval "$2=\$input1"
```

Comment:
one alternative here is to remove eval and read some variable called say `REPLY` and not have `$2` at all so it would look like. So that this variable would be accessible outside the function like so
```
prompt "Select episode: "
episode=$REPLY
```

Reply:
And that would be even less confusing, even though we need 2 replies because of episode ranges